### DISCUSSION TITLE
Error while generating geometry

### DISCUSSION THREAD

**Discussion by Ajaykrishna0703** [byuflowlab/FLOWUnsteady#172] [General]
Hi all
New user to FlowUnsteady, I am trying to simulate my own propeller in FlowUnsteady, I have generated all the required files for the simulation like propeller distributions and airfoils and i have added those files in rotor folder. While running the file i am getting a error which says "ERROR: LoadError: BoundsError: attempt to access 8×2 DataFrame at index [1, 3]" please help me in this.

**Comment by tashif-syed** [comment]
Hi
I faced a similar problem. The solution is as follows:
1. Your 'name_airfoils.csv' file might be having only 2 columns, 1st one being the Radius/R values and second being the contour file names. This might be the case since you generated the files yourself. If it is so then move on to next step
2. Create a 3rd column in that same file and name it any arbitrary name you want, example 'polar.csv'. Add the name for each row. This happens because the file structure required a polar file the be there.
3. In your code, make xfoil=True. It will then generate the actual files for your propeller blade
